acquire/acquire.py

```diff
@@ -557,6 +558,36 @@ def _run(cls, target: Target, cli_args: argparse.Namespace, collector: Collector
             mem_dump_errors_path.unlink()
 
 
+@register_module("--winmem-files")
+class WinMemFiles(Module):
+    DESC = "Windows memory files"
+    SPEC = [
+        ("file", "sysvol/pagefile.sys"),
+        ("file", "sysvol/hiberfil.sys"),
+        ("file", "sysvol/swapfile.sys"),
+        ("file", "sysvol/windows/memory.dmp"),
+        ("dir", "sysvol/windows/minidump"),
+    ]
+
+    @classmethod
+    def get_spec_additions(cls, target: Target) -> Iterator[tuple]:
+        spec = set()
+
+        resolver = ResolverPlugin(target)
```

Comment:
I believe you should just be able to do `target.resolve(path)` without having to instantiate the plugin yourself.